week-6/Homework/mandatory/2-quotegenerator/index.html

```diff
@@ -12,6 +12,22 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
+    <div class="jumbotron">
+      <h1 class="display-6">Get a random quote</h1>
+      <q class="lead" id="quote">Click the button or tick the box to get a quote</q>
+      <hr class="my-2">
+      <p id="author" class="lead">Author</p>
+      <p>
+        <a class="btn btn-primary" href="#" role="button" id="newbtn">New Quote</a>
+        
+      </p>
+      <p><label id="autolabel" class="form-check-label">
+          <input id="checkbox" class="form-check-input" type="checkbox" value="">
```

Comment:
It might be more idiomatic to close a single self-closing tag. But if you don't close it shouldn't cause any harm. My knowledge is staled on this subject, I have to make a research :)